beacon-chain/rpc/prysm/v1alpha1/beacon/validators.go

```diff
@@ -645,6 +642,10 @@ func (bs *Server) GetValidatorQueue(
 		exitQueueKeys[i] = vals[idx].PublicKey
 	}
 
+	churnLimit := helpers.ValidatorActivationChurnLimit(activeValidatorCount)
+	if headState.Version() >= version.Deneb {
+		churnLimit = helpers.ValidatorActivationChurnLimitDeneb(activeValidatorCount)
+	}
 	return &ethpb.ValidatorQueue{
 		ChurnLimit:                 churnLimit,
```

Comment:
This is a Prysm API implementation for streaming validator queue status. I doubt it's actively used. The return doesn't specify if the churn limit is for activation or exit; I assume it's for activation. Since this endpoint is rarely used and we're transitioning away from Prysm API, I don't see we need to further update this